src/main/java/com/eerussianguy/blazemap/api/builtin/TerrainSlopeMD.java

```diff
@@ -0,0 +1,28 @@
+package com.eerussianguy.blazemap.api.builtin;
+
+import java.util.Arrays;
+
+import com.eerussianguy.blazemap.api.BlazeRegistry;
+import com.eerussianguy.blazemap.api.pipeline.DataType;
+import com.eerussianguy.blazemap.api.pipeline.MasterDatum;
+
+public class TerrainSlopeMD extends MasterDatum {
+    private final BlazeRegistry.Key<DataType<MasterDatum>> id;
+    public final int[][] slopemap;
+
+    public TerrainSlopeMD(BlazeRegistry.Key<DataType<MasterDatum>> id, int[][] slopemap) {
+        this.id = id;
+        this.slopemap = slopemap;
+    }
+
+    @Override
+    public BlazeRegistry.Key<DataType<MasterDatum>> getID() {
+        return id;
```

Comment:
just return `BlazeMapReferences.MasterData.TERRAIN_SLOPE`
look at BlockColorMD and WaterDepthMD... you copied the one MD that does it wrong :p
either that or it needs the customization for some reason, if it doesn't you can rectify that one as well
the point is, this is not standard :)